src/views/who-to-tell.njk

```diff
@@ -27,7 +27,7 @@
 
       <h1 class="govuk-heading-xl">Who to tell about the company closing</h1>
       <p class="govuk-body">Directors must give a copy of this to all interested parties within 7 days of it being submitted.</p>
-      <p class="govuk-body">Read the <a href="{{ documentationUrl }}">full list of parties you must tell about the company closing (opens in a new tab)</a>.</p>
+      <p class="govuk-body">Read the <a class="govuk-link" href="{{ documentationUrl }}">full list of parties you must tell about the company closing (opens in a new tab)</a>.</p>
```

Comment:
I'm guessing that this isn't going to open in a new tab without `target="_blank"`

Reply:
Ahh, thanks ill set that in the commit